src/components/Searchbar/Search.tsx

```diff
@@ -17,6 +18,14 @@ export const Search = () => {
     staleTime: 1000 * 60 * 60 // 60 minutes caching
   });
 
+  const { data: newTitles } = useQuery({
```

Comment:
add key search to search anime by keywords, dont forget add it to keys list here, also you can implement some throttling to force calling api every time